src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileFormat.cs

```diff
@@ -509,11 +509,17 @@ private static LockFileTargetLibrary ReadTargetLibrary(string property, JToken j
         {
             var library = new LockFileTargetLibrary();
 
-            var parts = property.Split(new[] { '/' }, 2);
-            library.Name = parts[0];
-            if (parts.Length == 2)
+#pragma warning disable CA1307 // Specify StringComparison
+            int slashIndex = property.IndexOf('/');
```

Comment:
Why did you choose to suppress rather than fix the warning?

Reply:
There's no fix that works for all target frameworks.

Options were:

- different code by tfm
- use an overload that accepts string rather than char
- suppress warning 

I chose the last option as I don't think it applies to this character and the other options have downsides.

Reply:
Indeed, looks like they only added the overload for char in .NET Core. Note, this isn't about the character being searched, its what method they call underneath in response to the comparison. In this case, the default for `IndexOf` is ordinal (`Contains` is CurrentCulture!) so the behavior is okay.